src/store.js

```diff
@@ -7,6 +7,8 @@ var Store = module.exports = function(ctx) {
   this._features = {};
   this._featureIds = new StringSet();
   this._selectedFeatureIds = new StringSet();
+  this._selectedPoints = [];
+  this._selectedPointsFeatureId = null;
```

Comment:
I think we can drop this if we store the feature id as part of the selected coordinate properties